pkg/utils/utils_test.go

```diff
@@ -16,7 +16,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package ibm
+package log
```

Comment:
```suggestion
package utils
```

Not sure how this became `package log`, but could you also change this in `pkg/utils/utils.go`?

Reply:
Updated